src/components/Button/README.md

```diff
@@ -289,3 +289,120 @@ Use when a button has been pressed and the associated action is in progress.
 
 - To learn how to combine or lay out multiple buttons, [use the button group component](/components/actions/button-group)
 - To embed an action into a line of text, [use the link component](/components/navigation/link)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+### Structure
+
+Buttons can have different states that are visually and programmatically conveyed to merchants.
+
+- Use the `ariaControls` prop to add an `aria-controls` attribute to the button. Use it to point to the unique `id` of the content that the button manages.
+- If a button expands or collapses adjacent content, then use the `ariaExpanded` prop to add the `aria-expanded` attribute to the button. Set the value to convey the current expanded (`true`) or collapsed (`false`) state of the content.
+- Use the `disabled` prop to set the `disabled` state of the button. This prevents merchants from being able to interact with the button, and conveys its inactive state to assistive technologies.
+
+#### Navigation
+
+Merchants generally expect buttons to submit data or take action, and expect links to navigate. If you use the “url” prop to pass a value to the button, then the control outputs with an anchor styled as a button instead of an HTML button.
```

Comment:
```suggestion
Merchants generally expect buttons to submit data or take action, and expect links to navigate. If you use the `url` prop to pass a value to the button, then the control outputs with an anchor styled as a button instead of an HTML button.
```